src/components/pages/pages.test.js

```diff
@@ -130,37 +51,55 @@ context("Testing Pages component", () => {
     );
 
     it("should render Pages component and go next to Second page", () => {
-      CypressMountWithProviders(<PagesComponent />);
+      CypressMountWithProviders(<stories.PagesComponent />);
 
-      dataComponentButtonByText("Open Preview").click();
-      dataComponentButtonByText(`Go to second page`).click();
+      dataComponentButtonByText("Open Preview")
+        .click()
+        .then(() => {
+          dataComponentButtonByText("Go to second page").click();
+        });
 
       getDataElementByValue("title").should("have.text", "My Second Page");
     });
 
     it("should render Pages component and go next to Third page", () => {
-      CypressMountWithProviders(<PagesComponent initialpageIndex={1} />);
-
-      dataComponentButtonByText("Open Preview").click();
-      dataComponentButtonByText(`Go to third page`).click();
+      CypressMountWithProviders(
+        <stories.PagesComponent initialPageIndex={1} />
+      );
+
+      dataComponentButtonByText("Open Preview")
+        .click()
+        .then(() => {
+          dataComponentButtonByText("Go to third page").click();
+        });
 
       getDataElementByValue("title").should("have.text", "My Third Page");
     });
 
     it("should render Pages component and go back to Second page", () => {
-      CypressMountWithProviders(<PagesComponent initialpageIndex={2} />);
-
-      dataComponentButtonByText("Open Preview").click().wait(250);
-      backArrow().click();
+      CypressMountWithProviders(
+        <stories.PagesComponent initialPageIndex={2} />
+      );
+
+      dataComponentButtonByText("Open Preview")
+        .click()
```

Comment:
Is this now stable without the `wait(250)`?

Reply:
if you put the rest of the action in `.then` - yes